_summaries/goodfellow2013maxout_1.md

```diff
@@ -1,19 +1,92 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: goodfellow2013maxout
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: Zayne-sprague
+score: 8
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+*TODO - how is this different than max-pooling for CNNs?  Adding this would be helpful to the review.
+*TODO - add the visuals where marked
+*TODO - add equations where marked
+*TODO - break up sections a bit neater.
+
+
+**_Main Idea_**
+
+Taking the element-wise max of an input across multiple weight matrices produces a non-linear activation function
+that can learn any convex function and take advantage of dropout layers while improving accuracy and allowing deeper networks.
```

Comment:
I like this concise summary.  It describes exactly what maxout does, and provides a high-level intuition for why it works (i.e. approximating convex function as a max (which itself is a convex function) of piecewise linear functions learned by each weight matrix).

One minor point I have is that perhaps we can be a bit more descriptive about the expressivity of maxout.  It can _approximate_ any convex functions, but it certainly can't learn "any convex function", especially in practical setting where the number of weight matrices is limited.

Reply:
Good eye, adding that clarification as "learn approximations of any convex function".